lib/bolt/cli.rb

```diff
@@ -212,6 +214,15 @@ def print_help(mode)
       puts parser.help
     end
 
+    def parse_nodes(nodes)
+      list = get_arg_input(nodes)
+      if list == nodes
+        list.split(/[, ]+/)
+      else
+        list.split(/[\n]+/)
```

Comment:
Why avoid splitting on commas from a file? Don't like csv-style?

Reply:
I was thinking commas could appear in the URI and we wouldn't want to split on them, but the same is true for line 220. I could probably collapse the regex and always split on whitespace, `\n` or commas.

Reply:
puppet-task reads the file, then splits whatever list it ended up with on `[[:space:],]+`. So splitting on all whitespace and commas would be consistent with it.

Reply:
But ugh, commas are valid parts of a hostname. I'd make them consistent, and if it comes up we can do something to address it in both places.

Reply:
> Why avoid splitting on commas from a file? Don't like csv-style?

FWIW most unix tools are line-oriented and it's generally speaking more difficult to work with csv-style output.  The general use case I expect to work is something along the lines of:

    grep dev nodes.txt | bolt ...

Reply:
I tried using `[:space]`, but that didn't work:

```
irb(main):001:0> "  foo\nbar  \nbaz\n".split(/[[:space],]+/)
=> ["  foo\nb", "r  \nb", "z\n"]
```

So instead I went with `\s`, but not the first empty string:

```
irb(main):002:0> "  foo\nbar  \nbaz\n".split(/[\s,]+/)
=> ["", "foo", "bar", "baz"]
```

Hence the need to reject empty strings.